lib/processors/lib/utils.py

```diff
@@ -11,7 +12,11 @@ def get(self):
             userid = self.get_argument('userid')
             privatekey_pem = self.get_argument('privatekey', None)
             publickey_pem = self.get_argument('publickey', None)
-            user = User(userid, publickey_pem=publickey_pem, privatekey_pem=privatekey_pem)
+            user_blob = get_user(userid)
```

Comment:
So you're saying the user object should not include the user's name? Exhibits should store that themselves?

If that's the case, then we can gut everything about usernames out of the user object and handlers. This was just a misunderstanding of Peyman and myself about how we were updating things to include names.